src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -68,19 +73,22 @@ class PullRequestServices(client: BitbucketClient) {
     client.postJson(Request(url, classOf[JsObject]), JsNull)
   }
 
-  def createComment(author: String, repo: String, prId: Int, body: String,
+  def createLineComment(author: String, repo: String, prId: Int, body: String,
```

Comment:
If you want to specialise this method to be a line comment, the "file" and "lineTo" have to stop being optional.